src/patternfly/components/AppLauncher/app-launcher-menu-item-external-icon.hbs

```diff
@@ -1,6 +1,9 @@
-<i class="pf-c-app-launcher__menu-item-external-icon fas fa-external-link-alt{{#if app-launcher-menu-item-external-icon--modifier}} {{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
-  aria-hidden="true"
+<span class="pf-c-app-launcher__menu-item-external-icon{{#if app-launcher-menu-item-external-icon--modifier}}{{app-launcher-menu-item-external-icon--modifier}}{{/if}}"
   {{#if app-launcher-menu-item-external-icon--attribute}}
-	  {{{app-launcher-menu-item-external-icon--attribute}}}
-  {{/if}}></i>
+	    {{{app-launcher-menu-item-external-icon--attribute}}}
```

Comment:
Nit pick - can you update the indentation here so that there are 4 leading spaces, since we use 2 spaces for indentation and this line is indented 2 levels deep?

Reply:
Hey @ValerieKenyon looks like you pushed an update, but I'm still seeing too much indentation here https://github.com/patternfly/patternfly/blob/f52e914ad15e315bd98162a43dba5144cdd48e1b/src/patternfly/components/AppLauncher/app-launcher-menu-item-external-icon.hbs#L3

Reply:
I don't understand why that happened.  I did update it, and it showed indented 4 spaces in my VS Code.  I'm sorry about that.  I'll try to fix that now.

Reply:
I think I see where my mistake was.  The first 2 spaces in my file were from a tab, not the space bar.  I've fixed that and pushed the change, I think it looks okay now.  Again, sorry about that.

Reply:
Looks great now, thanks!